### PR TITLE
DynamoDB table should have backup enabled

### PR DESCRIPTION
**Pull request by pethers**
https://github.com/stelligent/cfn_nag/issues/463

### CODE REVIEW

**Comment by pethers** [comment]
@thegonch can you please review this ?

**Comment by pethers** [comment]
Can anyone at Stelligent please review this ?

**Review by arothian — APPROVED** [review]
Looks good, thanks for the contribution

File: lib/cfn-nag/custom_rules/DynamoDBBackupRule.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W78'
+  end
+
+  def audit_impl(cfn_model)
+    violating_ddb_tables = cfn_model.resources_by_type('AWS::DynamoDB::Table').select do |table|
+      table.pointInTimeRecoverySpecification.nil? || !truthy?(table.pointInTimeRecoverySpecification['PointInTimeRecoveryEnabled'].to_s)
```
Comment (thegonch) [inline lib/cfn-nag/custom_rules/DynamoDBBackupRule.rb:22]: This line will likely fail rubocop checks due to being too long.  Recommend changing to:

`table.pointInTimeRecoverySpecification.nil? ||`
`  ` `!truthy?(table.pointInTimeRecoverySpecification['PointInTimeRecoveryEnabled'].to_s)`

**Review by thegonch — CHANGES_REQUESTED** [review]
Looks good, just needs one update to pass rubocop.

**Comment by pethers** [comment]
> Looks good, just needs one update to pass rubocop.

Thanks for the feedback, will fix shortly.

**Review by thegonch — APPROVED** [review]

**Pull request merged**